superdesk/types/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .users import UsersResourceModel
```

Comment:
There is also a [superdesk.tests](https://github.com/superdesk/superdesk-core/blob/async/superdesk/types.py) file, which this module overrides causing an exception to be raised when running the app.

Could you move the content of that original file to this one, and delete the old one as well please